kong/conf_loader.lua

```diff
@@ -61,6 +61,8 @@ local CONF_INFERENCES = {
   cluster_advertise = {typ = "string"},
   nginx_worker_processes = {typ = "string"},
   upstream_keepalive = {typ = "number"},
+  server_tokens = {typ = "boolean"},
+  latency_tokens = {typ = "boolean"},
 
```

Comment:
@Tieske it is dabatable: http://nginx.org/en/docs/http/ngx_http_core_module.html#server_tokens